src/main/scala/org/encryfoundation/explorer/db/models/Output.scala

```diff
@@ -10,14 +11,16 @@ case class Output(id: String,
 object Output {
 
   import io.circe.Encoder
-  import io.circe.syntax._
+  import io.circe.generic.extras._
+
+  private implicit val config: Configuration = Configuration.default.copy(
```

Comment:
- more string of code
- plus 1 dependency
- purity of code - abstract argument